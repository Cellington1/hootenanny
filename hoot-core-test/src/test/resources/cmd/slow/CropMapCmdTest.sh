#!/bin/bash
set -e

mkdir -p $HOOT_HOME/test-output/cmd/slow/CropMapCmdTest

inputfile=$HOOT_HOME/test-files/DcGisRoads.osm
comparefile=$HOOT_HOME/hoot-core-test/src/test/resources/cmd/slow/CropMapCmdTest/cropped.osm
outputfile=$HOOT_HOME/test-output/cmd/slow/CropMapCmdTest/cropped.osm

# Run the command.
hoot crop-map $inputfile $outputfile -77.05,38.888,-77.03,38.90 

# Checking output
hoot is-match $comparefile $outputfile || diff $comparefile $outputfile
