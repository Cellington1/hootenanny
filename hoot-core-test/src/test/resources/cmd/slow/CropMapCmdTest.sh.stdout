INFO  ...p/hoot/core/ops/MapCropper.cpp( 115) Cropping map.
INFO  ...p/hoot/core/ops/MapCropper.cpp( 154)   Removing nodes...
INFO  ...late/SuperfluousWayRemover.cpp(  59) Removing superfluous ways...
INFO  ...ops/SuperfluousNodeRemover.cpp(  53) Removing superfluous nodes...
